src/modules/Lan.rb

```diff
@@ -1039,16 +1035,21 @@ def connected_and_bridgeable?(bridge_builder, interface)
       true
     end
 
+    # Refreshes YaST network configuration
+    #
+    # It does not modified the system configuration that was already read.
```

Comment:
is this really correct? maybe we need reread system configuration for e.g. s390 where new devices can appear or maybe when we need to load new kernel module for new interface? not sure.

Reply:
To be honest, I am not sure. We always work on the `yast_config` and use the `system_config` as a reference to decide whether we have something to write to the underlying system or not.

Reply:
yep, but s390 stuff we do immediatelly. So looks like our system_config is obsolete. But it will probably work, maybe just writter have some challenges.

Reply:
After discussing via IRC, we have found that what we want to do in this case is just refresh the list of interfaces, but keeping the rest of the configuration untouched.